### ISSUE TITLE
Portable man

### ISSUE THREAD

**Issue by ale5000-git** [rmyorston/busybox-w32#102]
Could you please make `man` searching manual pages also in a "man" folder near the busybox executable?
(in this way it is portable).

Like this:
```
c:\folder\busybox.exe
c:\folder\man\busybox.manpage
c:\folder\man\cat.manpage
c:\folder\man\grep.manpage
etc.
```
You don't need to provide the manpages, I find them myself.
Thanks in advance.

**Comment by rmyorston** [comment]
Good idea. I've added a 'man' directory in the same directory as the executable to the end of the list of paths to be searched.

Unless you have ```nroff``` and friends installed you'll need to put preprocessed (and optionally compressed) man pages in 'cat' directories like:
```
c:\folder\man\cat1\busybox.1.gz
c:\folder\man\cat1\cat.1
etc.
```

**Comment by rmyorston** [comment]
A new binary and its corresponding man page are now available on my website.

**Comment by ale5000-git** [comment]
Thanks, it now works but I don't understand the meanig of the cat1 folder and the 1 extension.
Is there any special meaning?

**Comment by rmyorston** [comment]
tl;dr: Historical Unix conventions.

The Unix manual is divided into sections. Section 1 covers commands, section 2 is system calls, section 3 is other functions, and so on. (This is why my commit messages sometimes refer to, e.g. ```kill(2)```, to indicate that I'm referring to the ```kill``` system call, not the ```kill``` command.)

My linux system keeps man pages in directories like:
```
/usr/share/man/man1
/usr/share/man/man2
/usr/share/man/man3
...
```
Within the directories there are files like ```kill.1.gz``` and ```kill.2.gz```.

These files are formatted as ```nroff``` source which has to be processed for display. As an optimisation it's possible to store preprocessed files:  these are kept in directories ```cat1``` and so on.

The ```man``` applet in BusyBox supports all of this complexity. In busybox-w32 we probably only want to have _preprocessed_ man pages for _commands_. Hence the use of the ```cat1``` folder and ```1``` extension.
